code/Model/SiteTree.php

```diff
@@ -2862,9 +2863,10 @@ public function getTreeTitle()
         $children = $this->creatableChildPages();
         $flags = $this->getStatusFlags();
         $treeTitle = sprintf(
-            "<span class=\"jstree-pageicon page-icon %s class-%s\"></span><span class=\"item\" data-allowedchildren=\"%s\">%s</span>",
+            "<span class=\"jstree-pageicon page-icon %s class-%s%s\"></span><span class=\"item\" data-allowedchildren=\"%s\">%s</span>",
             $this->getIconClass(),
             Convert::raw2htmlid(static::class),
+            $this->isHomePage() ? " homepage" : "",
```

Comment:
Can you use single quotes here, since you've used them in the rest of your code?